.changes/unreleased/tinymce-TINY-11692-2025-06-17.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Deprecated
+body: Deprecated 'setContent' method in EditorSelection class.
```

I think the EditorSelection is a bit internal so maybe we change the wording so that it points to how a user would likely call this.

```suggestion
body: Deprecated 'editor.selection.setContent' method.
```